boards/sparkfun/artemis_nano/Kconfig.artemis_nano

```diff
@@ -0,0 +1,10 @@
+# Sparkfun artemis nano board configuration
+# SPDX-License-Identifier: Apache-2.0
+
+
+config BOARD_ARTEMIS_NANO
+	select SOC_APOLLO3_BLUE
+
+config FLASH_BASE_ADDRESS
+    hex
+    default 0x10000
```

Comment:
There are missing newlines in some files :)

Reply:
Could you please explain it more for me. What do you mean by "missing newlines"?

Reply:
Files need to end with a newline, in most IDEs, the end of a file should look like
```
132 |         foo();
133 | }
134
```
You can see the files missing a newline if you review this PR on github https://github.com/zephyrproject-rtos/zephyr/pull/75210/files